libs/image-utils/src/image_data.test.ts

```diff
@@ -300,3 +301,24 @@ test('ensureImageData', () => {
   expect(ensureImageData(imageDataLike) === imageDataLike).toBeFalsy();
   expect(ensureImageData(imageDataLike)).toBeInstanceOf(ImageData);
 });
+
+test('toImageBuffer', async () => {
+  await fc.assert(
+    fc.asyncProperty(
+      arbitraryImageData({ width: 5, height: 5, channels: 4 }),
+      fc.constantFrom<'png' | 'jpeg' | undefined>('png', 'jpeg', undefined),
+      async (imageData, format) => {
+        const buffer = toImageBuffer(imageData, format && `image/${format}`);
+        const filePath = fileSync({ template: `tmp-XXXXXX.${format}` }).name;
```

Comment:
Clean these up on test end?

Reply:
I think they are cleaned up automatically? The docs for `fileSync` say:
> Simple synchronous temporary file creation, the file will be closed and unlinked on process exit.

Reply:
The other tests in this file also seem to trust the automatic cleanup